src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -53,6 +54,8 @@ class Submitter(stagingBaseDir: File,
       case NonFatal(e) => throw new IOException(s"unexpected error occured on $stagingBaseDir", e)
     }
   }
+  val srcProvider: FileSystemProvider = stagingBaseDir.fileSystem.provider()
```

Comment:
how is this `stagingBaseDir.fileSystem.provider()` different from the `stagingBaseDir.path.getFileSystem` above?

Reply:
overwrite false is a good idea, otherwise better safe than sorry with a double check than rely on configuration checks.